### DISCUSSION TITLE
XDrip+ Webservice stops when phone screen is off

### DISCUSSION THREAD

**Discussion by renegadeandy** [NightscoutFoundation/xDrip#3060] [General]
Hi everyone. I have enabled the Web service for my watch and it only appears to work when the phone is in use. 

If I let me phones screen turn off....the watch never updates its bg reading. I turn on the screen and boom, BG is updated  on the watch. Is there any setting I can check to see if there is a problem with the way xdrip is configured?

**Comment by renegadeandy** [comment]
I may have fixed this by going into the Android settings, into the app list, choosing xdrip+, selecting battery, and allowing the app to be Unrestricted instead of Optimised.

**Reply by Navid200** [reply]
Thanks

**Comment by jamorham** [comment]
This will be handset specific so please tell us which phone manufacturer and model this is.
